tools/utils/vrbopen.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+URL=$1
+VRB_EXT=$2
+VRB_APP=org.mozilla.vrbrowser
+WAVEVR_EXT=wavevr
+DEV_EXT=dev
+
+if [ -z "$VRB_EXT" -o "$VRB_EXT" = "$WAVEVR_EXT" ] ; then
+  VRBINSTALLED=`adb shell pm list packages $VRB_APP.$WAVEVR_EXT`
+  if [ ! -z "$VRBINSTALLED" ] ; then
+    VRB_EXT=.$WAVEVR_EXT
+  elif [ -n "$VRB_EXT" ] ; then
+    echo Error: $VRB_APP.$WAVEVR_EXT not installed.
+    exit 1
+  fi
+fi
+
+if [ -z "$VRB_EXT" -o "$VRB_EXT" = "$DEV_EXT" ] ; then
+  VRBINSTALLED=`adb shell pm list packages $VRB_APP.$DEV_EXT`
+  if [ ! -z "$VRBINSTALLED" ] ; then
+    VRB_EXT=.$DEV_EXT
+  elif [ -n "$VRB_EXT" ] ; then
+    echo Error: $VRB_APP.$DEV_EXT not installed.
+    exit 1
+  fi
+fi
+
+if [ "$VRB_EXT" == "default" ] ; then
+  VRB_EXT=""
+fi
+echo Using $VRB_APP$VRB_EXT
+
+if [ "$URL" = x ] ; then
```

Comment:
I'm thinking if it should be `"X$URL" = "X"` if you would like to check if $URL is null  or empty

Reply:
If it is null there is no need to check. I can add quotes around `x` for consistency. I might also be able to get rid of the `x` entirely.